source/_integrations/griddy.markdown

```diff
@@ -15,7 +15,7 @@ The `griddy` integration allows you to integrate your [Griddy](https://griddy.co
 
 There is currently support for the following device types within Home Assistant:
 
-- Sensor
+- Energy
 
```

Comment:
Please revert this, as it describes device platform types, not category.

Reply:
Whoops...got edit happy.